x-pack/build_chromium/build.py

```diff
@@ -38,6 +38,7 @@
   raise Exception('Unexpected architecture: ' + arch_name + '. `x64` and `arm64` are supported.')
 
 print('Fetching locale files')
+# TODO: move this into the repo itself, so we are only writing the build output to the bucket
```

Comment:
Does this comment mean that we should use `.buildkite/scripts/pipelines/chromium_linux_build/build.sh` for the operation to copy the build output to staging?

Reply:
This was a consideration for not pulling the locale from gcloud, so we have all the required portions for the build contained within the repo, and only having build outputs in storage, we don't have to do this if it doesn't make sense, what do you think?

Reply:
++ I think it makes sense to have the required locale portions contained within the repo. This seems like something that can wait until later, so I guess having a `TODO` is fine here. :)